header.php

```diff
@@ -56,6 +56,70 @@
   <section id="main-container">
 
   <!-- start content -->
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+  <header id="header" class="u-fixed">
+    <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+    <nav class="menu">
+      <ul class="main-menu row">
+<?php 
+$posts = get_posts();
+
+if (count($posts) > 0) {
+?>
+        <li>
+          Studio
+          <ul class="sub-menu">
+<?php 
+  foreach($posts as $post) {
+    setup_postdata($post);
+?>
+            <li class="studio-list-item">
+              <a href="<?php the_permalink(); ?>"><?php the_title(); ?></a>
+            </li>
+<?php 
+  }
+  wp_reset_postdata();
+?>
+          </ul>
+        </li>
+<?php 
+}
+
+$cat_array = get_categories();
+
+if ($cat_array) {
+?>
+        <li>
+          Sort
```

Comment:
not just styling tho cus I think the markup needs to be different. But its a question of where the line is between scaffold and styling. imo scaffold should be more or less functional both in the sense of what the markup actually does but usable. This feature is a good example as its quite hard to test the feel of the UX as the header is unstyled other than the default `.u-fixed`. I've been adding z-index just to test so it might as well be in the feature branch imo. I don't think scaffold should prohibit styling but it shouldn't worry about detailing just functional and prefering already existing css rules and classes.